language-design.md

```diff
@@ -17,7 +12,41 @@ For each quote, describe what it was about the quote that led you pick it.
 
+> If your standard is natural language, division/fraction notation isn't particularly intuitive.
+> But to someone who knows the notation, ab / b is very intuitive, and it's immediately obvious that
+> you can cancel the bs. So the goal of an intuitive programming syntax shouldn't be to emulate
+> natural language, necessarily, but to provide intuitive hooks and syntax for the types of things
+> that programmers do a lot. 
+> [Comment by "Peter" ([Source](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-bc08f700-3c06-11e1-921b-9794fc737ae8))] [Pavlus, 2012]
+
+I feel that this comment on the Pavlus article really gets to the heart of the matter - that knowing the notation is part of any (technical) job, and once you can grok it, it becomes second nature.  So while natural language may make it easier to read and write code at first, when it comes down to it the most important thing is making sure that someone who uses it often (i.e. professionals and very passionate amateurs) can do what they need to do clearly and easily.
+
+The other point is that often, natural language is *not* actually suitable for certain purposes. Many notations are much *easier* than natural language. Usability is indeed a good aspiration, but the answer isn't so simple as "natural language". In fact, there probably isn't just one good answer. Hence why we are in a class called domain-specific languages!
+
+> **Example code should be exemplary.**  
+> If an API is used widely, its examples will be the archetypes 
+> for thousands of programs. Any mistakes will come back to haunt 
+> you a thousand fold. [Bloch, 2006]
+
+This is perhaps my favorite quote of the entire reading.  Having spent this past summer working with a horrible API (most modules were not documented, and only 23% of the examples worked (we checked)) I appreciate and understand this the most.  Your API is how to get developers to use your project.  If you make a large barrier to entry in the form of missing or bad documentation and examples, they will look elsewhere for their solutions. 
+
+Meanwhile, bad examples are also indicative of other problems. To make a good language, you should have a good idea of how it should be used. Furthermore, it should be useful in the sense that "the way it should be used" should be *good*. If you find yourself writing poor examples, that may mean that the easiest thing to do in your framework is write bad code, and that you may not have put enough time into considering how you intend your work to be used.
```

Comment:
I completely agree, I think this goes with Bloch's other point about performance and boilerplate code. If an API is designed well, the intuitive implementation will be performance optimized and the user probably won't write boilerplate code.